src/App.tsx

```diff
@@ -1,24 +1,201 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState, useMemo } from 'react';
+import classNames from 'classnames';
+
 import { UserWarning } from './UserWarning';
+import { TodoList } from './components/TodoList/TodoList';
+import { Todo } from './types/Todo';
+import {
+  createTodo, getTodos, removeTodo, updatingTodo,
+}
+  from './api/todos';
+import { Filters } from './types/enumFilter';
+import { Filter } from './components/Filter/Filter';
+// eslint-disable-next-line
+import { ErrorNotification } from './components/Notification/ErrorNotification';
+import { prepareTodos } from './utils/prepareTodos';
+import { AddTodoForm } from './components/AddTodoForm';
 
-const USER_ID = 0;
+const USER_ID = 11241;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filteringField, setFilteringField] = useState(Filters.All);
+  const [errorMessage, setErrorMessage] = useState<string>('');
+  const [loading, setLoading] = useState(false);
+  const [tempTodo, setTempTodo] = useState<null | Todo>(null);
+  const [loadingTodoIds, setLoadingTodoIds] = useState<number[]>([]);
+  const someCompleted = todos.some(todo => todo.completed);
+  const everyCompleted = todos.every(todo => todo.completed);
+
+  useEffect(() => {
+    getTodos(USER_ID)
+      .then(setTodos)
+      .catch(() => {
+        setErrorMessage('Unable to load a todo');
+      });
+  }, []);
+
+  const visibletodos = useMemo(
+    () => prepareTodos(todos, filteringField), [todos, filteringField],
+  );
+
   if (!USER_ID) {
     return <UserWarning />;
   }
```

Comment:
I think it should be placed just before the main return and not between some functions